### PR TITLE
created contracts, notifications screen and proposal screen

### PR DESCRIPTION
**Pull request by Manan-Sharma-5**
## Summary by CodeRabbit

- **New Features**
	- Introduced a dedicated Notifications screen for better user engagement.
	- Enhanced navigation with clearer tab titles: "Proposals" and "Contracts."
	- Added interactive components for displaying contract and proposal details.
	- Implemented a dynamic notification card interface.

- **Bug Fixes**
	- Improved header styles for better consistency and readability.
	- Adjusted rendering logic to prevent potential runtime errors in components.

- **Style**
	- Updated styles for various components to enhance visual appeal and responsiveness.

- **Documentation**
	- Added new interfaces and constants for better data structure clarity.

### CODE REVIEW

**Comment by Manan-Sharma-5** [comment]
Screenshot 2024-09-09 at 3 48 10 PM
Screenshot 2024-09-09 at 3 48 02 PM
Screenshot 2024-09-09 at 3 47 56 PM
Screenshot 2024-09-09 at 3 47 48 PM
Screenshot 2024-09-09 at 3 47 39 PM
Screenshot 2024-09-09 at 11 58 15 AM

File: constants/Contracts.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
+interface ProposalCard {
```
Comment (dev-rathore) [inline constants/Contracts.ts:1]: I think this will be in types/Contracts.ts

File: components/NotificationCard.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+export const NotificationCard = ({
+  adName,
+  userName,
+  brandName,
+  notification,
+  avatar,
+  time,
+  action,
+}: {
+  adName: string;
```
Comment (dev-rathore) [inline components/NotificationCard.tsx:16]: This can be converted into a type.

File: app/(main)/(drawer)/(tabs)/_layout.tsx [hunk 3]
```diff
@@ -44,11 +44,11 @@ const TabLayout = () => {
           },
           tabBarIcon: ({ color }) => <TabBarIcon name="home" color={color} />,
           headerRight: () => (
-            <Link href="/modal" asChild>
+            <Link href="/notification-screen" asChild>
```
Comment (dev-rathore) [inline app/(main)/(drawer)/(tabs)/_layout.tsx:47]: We might have conflicts since I have named it as /notification

**Review by dev-rathore — COMMENTED** [review]
Looks good to me, you can look for three comments.

Also I would like you to match the app folder files, (drawer)/(tabs) and (screens) so that we can have consistent routes and filename.
https://github.com/idivarts/trendly-users/pull/9

**Pull request closed without merging**